tests/e2e/verfication_post_day1_test.go

```diff
@@ -61,22 +78,34 @@ var _ = Describe("TF Test", func() {
 				clusterAvailZones := getResp.Body().Nodes().AvailabilityZones()
 				Expect(err).ToNot(HaveOccurred())
 				Expect(getResp.Body().MultiAZ()).To(Equal(profile.MultiAZ))
-				if clusterAvailZones != nil {
+
+				if profile.Zones != "" {
 					Expect(clusterAvailZones).
 						To(Equal(H.JoinStringWithArray(profile.Region, zonesArray)))
-				} else {
-					Expect(clusterAvailZones).To(Equal(nil))
 				}
 			})
 		})
 		Context("Author:smiron-High-OCP-68423 @OCP-68423 @smiron", func() {
 			It("Verify compute_labels are set post cluster creation", CI.Day1Post, CI.High, func() {
 				getResp, err := CMS.RetrieveClusterDetail(CI.RHCSConnection, clusterID)
 				Expect(err).ToNot(HaveOccurred())
+
 				if profile.Labeling {
 					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(CON.DefaultMPLabels))
 				} else {
-					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(""))
+					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(CON.NilMap))
+				}
+			})
+		})
+		Context("Author:smiron-High-OCP-63777 @OCP-63777 @smiron", func() {
+			It("Verify AWS tags are set post cluster deployment", CI.Day1Post, CI.High, func() {
+				getResp, err := CMS.RetrieveClusterDetail(CI.RHCSConnection, clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				if profile.Tagging {
+					Expect(getResp.Body().AWS().Tags()["tag1"]).To(Equal(CON.Tags["tag1"]))
+				} else {
+					Expect(getResp.Body().AWS().Tags()["tag1"]).To(Equal(""))
```

Comment:
@shellymiron did you verify non-tagging scenario? I guess tag1 shouldn't be found or return null/error

Reply:
yeah, checkout cluster `export CLUSTER_ID=27c2i17u3f3i02kub55ng4639en0c6ed`       
<img width="1431" alt="image" src="https://github.com/terraform-redhat/terraform-provider-rhcs/assets/54883783/3eff8b5f-0d37-44a4-9534-f4ce1c4ed08a">

Reply:
I think that in golang if a map's key is not found, it just return empty string, that's why the test worked